src/components/MemberDescription.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MemberDescription = ({
+    name,
+    role,
+    img,
+    education,
+    companies,
+    bio,
+    speaking,
+}) => {
+    return (
+        <div className="overflow-hidden bg-white">
+            <div className="relative mx-auto max-w-6xl py-14 px-4 sm:px-6 lg:px-8">
+                <div className="mt-7 lg:grid gap-0 lg:grid-cols-2">
+                    <div className="relative ml-8 mt-4 lg:col-start-1 lg:row-start-1">
+                        <div className="relative mx-auto max-w-prose text-base lg:max-w-none">
+                            <figure>
+                                <img
+                                    className="rounded-lg object-cover object-center shadow-lg"
+                                    src={img}
+                                    alt="Whitney leaning against a railing on a downtown street"
+                                    width={320}
+                                    height={300} //TODO[CHRIS FROM YSSA]: need to edit image sizes
+                                />
+                            </figure>
+                        </div>
+                        <div className="mx-auto max-w-prose text-base lg:grid lg:max-w-none lg:grid-cols-2 lg:gap-8">
+                            <div>
+                                <h3 className="mt-2 font-bold text-faster-green sm:text-2xl">
+                                    {name}
+                                </h3>
+                                <h4>{role}</h4>
+                                <h4 className="mt-5 font-bold text-faster-green">
+                                    EDUCATION
+                                </h4>
+                                <h5>
+                                    {education?.map((school) => (
+                                        <h5 key={school}>{school}</h5>
+                                    ))}
+                                </h5>
+                                <h4 className="mt-2 font-bold text-faster-green">
+                                    COMPANIES
+                                </h4>
+                                {companies?.map((company) => (
+                                    <h5 key={company}>{company}</h5>
+                                ))}
+                            </div>
+                        </div>
+                    </div>
+                    <div className="mr-8 -ml-24 lg:mt-0">
```

Comment:
ah I think for this we may have to add some extra breakpoint at smaller viewports. It's causing text to go too far left at smaller breakpoints

Reply:
Fixed!